packages/cli/src/commands/init/getDirectoryFilesRecursive.ts

```diff
@@ -0,0 +1,33 @@
+import fs from 'fs';
+import path from 'path';
+import {promisify} from 'util';
+
+const readdir = promisify(fs.readdir);
+
+export async function getDirectoryFilesRecursive(
```

Comment:
How about executing a simple `find ${dir} -print` command? For example, running `find ios -print` on my React Native project gave me the following:
```
ios
ios/react-native-hello-world-generated.mm
ios/HelloWorld.h
ios/HelloWorld.mm
ios/react-native-hello-world.h
ios/HelloWorld.xcodeproj
ios/HelloWorld.xcodeproj/project.pbxproj
```

Reply:
I would like to simplify this function as it looks a bit complex (while it certainly gets the job done).